test/parallel/test-postmortem-metadata.js

```diff
@@ -26,6 +26,11 @@ const nm = spawnSync('nm', args);
 if (nm.error && nm.error.errno === 'ENOENT')
   common.skip('nm not found on system');
 
+const stderr = nm.stderr.toString();
+if (nm.stderr.toString().length > 0) {
```

Comment:
Nit: can't `stderr` be used here instead of repeating `nm.stderr.toString()`?

Reply:
@Ipinca oops, somehow I forgot to replace this...thanks

Reply:
Ping @joyeecheung